requirements.txt

```diff
@@ -10,3 +10,10 @@ pytest-json==0.4.0
 pytest-metadata==1.5.1
 pytest==3.3.2
 python-dateutil==2.6.1
+git+https://github.com/hwine/python-herokuadmintools.git#egg=herokuadmintools
+wheel==0.33.4
+# We want pyjq==2.3.1, but it can't be built in the container (we don't
+# want to load all the compilers, etc). So assume the wheel is prebuilt
+# in the RESULTS_DIR, which requires pip >= 10 per
+#   https://pip.pypa.io/en/stable/reference/pip_install/#using-environment-variables
+${RESULTS_DIR}/pyjq-2.3.1-cp36-cp36m-linux_x86_64.whl
```

Comment:
I'm not a fan of this, makes it really hard for people to use outside of our team.

I'd say we load the compilers in our container

Reply:
Let's discuss. I don't believe in leaving compilers in production images. We could do it with multi-stage docker images, but that's harder to maintain.

And, based on my experience, this code is really hard to reuse without documentation, so I'm not sure we can achieve purity here.

Reply:
as discussed, should just install pyjq